src/app/components/Image/index.stories.tsx

```diff
@@ -42,16 +42,21 @@ export const ResponsiveWebPWithJpegFallback = () => (
 );
 
 export const BasicAMPImage = () => (
-  <Image
-    isAmp
-    alt="A penguin stands on an ice floe"
-    src="https://ichef.bbci.co.uk/ace/ws/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
-  />
+  <RequestContextProvider
+    isApp
+    pageType="article"
+    pathname=""
+    service="afaanoromoo"
```

Comment:
Let's use the default values and just set the values we need (unless it causes a visual difference)
```suggestion
    isAmp
```